screen/budgets/src/main/java/com/ivy/budgets/BudgetScreen.kt

```diff
@@ -244,6 +248,25 @@ private fun Toolbar(
                         fontWeight = FontWeight.ExtraBold
                     )
                 )
+
+                val totalRemainingText = when {
+                    categoryBudgetText.isNotEmpty() -> stringResource(
+                        if (budgetExceeded) R.string.budget_exceeded_info else R.string.total_budget_info,
+                        abs(totalRemainingBudget).format(baseCurrency),
+                        baseCurrency
+                    )
+                    else -> ""
+                }
+
+                if (categoryBudgetText.isNotEmpty()) {
```

Comment:
Is this `if` correct? I guess we need to check `totalRemaingText` not being null